Source/Common/GLTFSceneSource.swift

```diff
@@ -25,19 +25,20 @@ public class GLTFSceneSource : SCNSceneSource {
     }
     
     public override convenience init(url: URL, options: [SCNSceneSource.LoadingOption : Any]? = nil) {
```

Comment:
? - do we want this to throw as well?

Reply:
Not sure if we use this though

Reply:
That is the init we currently use in the app. However, that is an overridden init so we cannot make it throw since that would modify the signature.

Reply:
We have modified the app to use the other init that throws.